packages-tests/FileFormatter/ValueObject/NewLineTest.php

```diff
@@ -6,10 +6,13 @@
 use PHPUnit\Framework\TestCase;
 use Rector\FileFormatter\Exception\InvalidNewLineStringException;
 use Rector\FileFormatter\ValueObject\NewLine;
+use Rector\Testing\PHPUnit\PlatformAgnosticAssertions;
 use Symplify\SmartFileSystem\SmartFileInfo;
 
 final class NewLineTest extends TestCase
 {
+    use PlatformAgnosticAssertions;
```

Comment:
this line marks this class for a PHP_EOL independent `assertSame`. it makes this testcase pass on windows and linux.

without this line, it fails on windows:

```
2) Rector\Tests\FileFormatter\ValueObject\NewLineTest::testFromFiles with data set "Xml file with line feed" (Symplify\SmartFileSystem\SmartFileInfo Object (...), '\n')
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
 #Warning: Strings contain different line endings!
-'
+'
 '
```